site/releases/2024-dec-19/release-notes.qmd

```diff
@@ -0,0 +1,411 @@
+---
+title: "December 19, 2024"
+---
+
+This release brings additional customization features to the {{< var validmind.platform >}}, improvements to our documentation site, a brand new Help Center, and more!
+
+::: {.highlights}
+
+## {{< fa bullhorn >}} Release highlights
+
+### {{< var validmind.platform >}} (v1.27.17)
+
+<!---
+PR #1079: Added the ability to Managed Saved Views
+URL: https://github.com/validmind/frontend/pull/1079
+Labels: enhancement
+--->
+#### Custom model inventory and findings views
+<span id="saved-views"></span>
+Along with customizing what you see, you can now save preconfigured views on the **{{< fa cubes >}} Inventory** and all **{{< fa triangle-exclamation >}} Findings** pages within {{< var validmind.platform >}}, allowing you to retrieve these settings easily for future use.
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-40-ns .tc}
+[Customize model inventory layout](/guide/model-inventory/customize-model-inventory-layout.qmd){.button .button-green}
+
+:::
+
+::: {.w-40-ns .tc}
+[View and filter model findings](/guide/model-validation/view-filter-model-findings.qmd#customize-model-findings-layout){.button .button-green}
+
+:::
+
+::::
+
+<!-- MANUAL ADD
+https://github.com/validmind/documentation/pull/583 -->
+
+#### Custom dashboards and widgets
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-60-ns}
+We've added the capability to build custom **{{< fa home >}} Dashboard**s to allow quick access to information relevant to you via widgets.
+
+:::
+
+::: {.w-40-ns .tc}
+[Customize your dashboard](/guide/configuration/customize-your-dashboard.qmd){.button .button-green}
+
+:::
+
+::::
+
+Widget types include saved inventory and model findings views[^1], analytics reports,[^2] and model activity.[^3] 
+
+![Custom `High Risk` dashboard showing widgets for Tier 1 models and High Severity findings](custom-dashboard.png){width=80% fig-alt="A screenshot of a custom `High Risk` dashboard showing widgets for Tier 1 models and High Severity findings" .screenshot}
+
+### Documentation (v1.27.17)
```

Comment:
> Mislabelled since we also have a documentation section further down (which covers just docs site and related updates). This section is for product highlights for the platform and library.*

We did some documentation highlights in the December 6, 2024 release notes to too to highlight our cool docs features — the duplicate sections here are deliberate. My opinion is that it should stay.

Reply:
Additional comment: Especially as we don't do these often — I've just done them in the last two because we NEVER highlighted cool docs stuff!

Reply:
I love that sentiment, @validbeck, even if it feels a bit like the docs are taking over the product. 😁 That said, I still think we need to have the heading be for the ValidMind Library here — it's part of the core product features. If you want to squeeze in some highlights from our team below that, I think that might be a reasonable compromise.

Reply:
There AREN'T any core ValidMind Library updates for this release is what I'm saying — the code samples are technically part of the documentation (and we've labelled it as documentation releases in the past).

Reply:
> There AREN'T any core ValidMind Library updates for this release is what I'm saying — the code samples are technically part of the documentation

Code samples live in the validmind-library repo for a reason. Yes, we build them as part of our documentation but that's not how they can be consumed effectively. You need to run code samples locally or on JupyterHub and they depend on a version of the ValidMind Library to run. Typically, these new or updated notebooks highlight library features that are being introduced in this version of the library — whatever we have in the docs site is a static copy of the real deal, though these notebooks are a literate programming tool. That makes them different from our other documentation in important ways.

I think the minimum changes needed here are:

1. Switch the heading as suggested
2. Separate the user guide updates from the Jupyter Notebooks updates